akka-docs/src/main/paradox/index-cluster.md

```diff
@@ -0,0 +1,17 @@
+# Clustering
+
+@@toc { depth=2 }
+
+@@@ index
+
+* [common/cluster](common/cluster.md)
```

Comment:
I wonder how interesting this Cluster Specification is for end users? Does it deserve a place in the top toc or should we link to it from somewhere inside cluster-usage.md?